<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BluetoothForm</class>
 <widget class="QWidget" name="BluetoothForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Scan for Devices</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="status">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QListWidget" name="lastList"/>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="deviceList">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="preview">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
